.goreleaser.yml

```diff
@@ -30,7 +37,14 @@ builds:
       - -v
       - -trimpath
     ldflags:
-      - -s -w -X github.com/ethersphere/bee.version={{.Version}} -X github.com/ethersphere/bee.commitHash={{.ShortCommit}} -X github.com/ethersphere/bee.commitTime={{.CommitTimestamp}}
+      - -s -w
+      - -X github.com/ethersphere/bee.version={{.Version}}
+      - -X github.com/ethersphere/bee.commitHash={{.ShortCommit}}
+      - -X github.com/ethersphere/bee.commitTime={{.CommitTimestamp}}
+      - -X github.com/ethersphere/bee/pkg/api.Version={{.BeeApiVersion}}
+      - -X github.com/ethersphere/bee/pkg/debugapi.Version={{.BeeDebugApiVersion}}
+      - -X github.com/ethersphere/bee/pkg/p2p/libp2p.reachabilityOverridePublic={{.ReachabilityOverridePublic}}
```

Comment:
also this. the build process should not even be aware we are providing these overrides

Reply:
Removed.

Reply:
@acud they should be inside [Makefile](https://github.com/ethersphere/bee/blame/38787a97c52fc03de6f521caff552964bb4f9496/Makefile#L32)?